static/app/components/gridEditable/index.tsx

```diff
@@ -403,11 +433,3 @@ class GridEditable<
 }
 
 export default GridEditable;
-export {
-  COL_WIDTH_MINIMUM,
-  COL_WIDTH_UNDEFINED,
-  GridColumn,
-  GridColumnHeader,
-  GridColumnOrder,
-  GridColumnSortBy,
```

Comment:
This re-export was confusing webpack and causing warnings (not errors, just a few weird warnings).

Reply:
yes I noticed the warning another day